### ISSUE TITLE
Ability to set text alignment

### ISSUE THREAD

**Issue by adrian5** [synfig/synfig#302]
Currently there appears to be no way to specify whether (multi-line) text should be left, center, or right-aligned. The behavior defaults to center-aligned, which makes multi-line text fields fairly pointless.

**Comment by caryoscelus** [comment]
As far as i know, text support is generally pretty poor.

**Comment by adrian5** [comment]
I see, thanks for the heads up! I had hoped to use Synfig to create lively tutorials involving diagrams and textual content. I'll see if importing text as an SVG file is an option for me.

p.s. the title heading in README.md reads _Synig_ Studio.

**Comment by caryoscelus** [comment]
Thanks, fixed that.

**Comment by blackwarthog** [comment]
> Currently there appears to be no way to specify whether (multi-line) text should be left, center, or right-aligned. The behavior defaults to center-aligned, which makes multi-line text fields fairly pointless.

You can use 'orientation' property.
Try values:
  0, 0
  1, 1
  1, 0.5
  ...
You will see how it works

--
Ivan

**Comment by adrian5** [comment]
Yes, (0, 0) works for left-alignment, thanks. I didn't even try that field, because the values are designated as pixel units, which wasn't suggestive of alignment.

I would like to keep this ticket open as a feature/improvement request for the future, since a menu for `Left, Center, Right, Justify-*` would still be convenient.

**Comment by morevnaproject** [comment]
> Try values: 
> 0, 0
> 1, 1
> 1, 0.5

For that values to work you need to convert "Orientation" parameter to "Composite":
![screen-12](https://cloud.githubusercontent.com/assets/332868/22334439/63bf4766-e40d-11e6-96a8-3acaacbdbae7.png)
